libs/shared/ui/tsconfig.storybook.json

```diff
@@ -5,27 +5,10 @@
     "outDir": ""
   },
   "files": [
-    "../../node_modules/@nx/react/typings/styled-jsx.d.ts",
+    "../../../node_modules/@nx/react/typings/styled-jsx.d.ts",
```

Comment:
Maybe a mistake it's weird to have a different path compared with the bottom paths, no? 

If it's generate by Nx ok for me 👍

Reply:
It doesn't seems to have impact on storybook but it's weird you're right. I'll check on fresh project

Reply:
Yes I tried the Storybook locally and it works

Reply:
Have align the other `d.ts` this is what is generated by Nx from scratch
https://github.com/Qovery/console/pull/921/commits/a33b0aa5f5c14bfb50c539deead2543eab673066